packages/jokul/package.json

```diff
@@ -510,6 +513,7 @@
         "nanoid": "^3.3.6",
         "postcss": "^8.4.24",
         "react-hook-form": "^7.44.3",
+        "react-is": "^18.0.0",
```

Comment:
Skal denne være `devDependencies` i stedet for `dependencies`? Brukes den ikke i produksjon?

Reply:
Nei, vite vil i løpet av bygget lage en kopi av den og ta den med i artifactet siden den ligger i lista over ting som er excluded i node-externals pluginen. Konsumentene trenger derfor ikke å installere den selv.

Reply:
Aha, ikke vært borti den exclude saken før